daterangepicker.js

```diff
@@ -1394,6 +1417,14 @@
             this.container.remove();
             this.element.off('.js-daterangepicker');
             this.element.removeData();
+        },
+
+        setTimeForCustomRange: function() {
+            this.setStartDate(this.startDate.startOf('day'));
```

Comment:
Does this imply that it's being passed a moment date object.   Is that part of the library?